ginivision/src/main/java/net/gini/android/vision/internal/camera/photo/Photo.java

```diff
@@ -81,17 +115,23 @@ public synchronized int getRotationForDisplay() {
         return mRotationForDisplay;
     }
 
-    public synchronized void setRotationForDisplay(int degrees) {
-        // Converts input degrees to degrees between [0,360]
+    synchronized void setRotationForDisplay(int degrees) {
+        // Converts input degrees to degrees between [0,360)
         mRotationForDisplay = ((degrees % 360) + 360) % 360;
     }
 
-    public synchronized void setRotationDelta(int degrees) {
-        // Converts input degrees to degrees between [0,360]
-        mRotationDelta = ((degrees % 360) + 360) % 360;
+    synchronized void updateRotationDeltaBy(int degrees) {
```

Comment:
Renamed, to make the intent clear, that the rotation delta change is cumulative.